packages/remix-dev/__tests__/cli-test.ts

```diff
@@ -236,12 +236,23 @@ describe("remix cli", () => {
           path.join(__dirname, "fixtures", "arc.tar.gz")
         ).toString(),
         "--no-install",
+        "--no-typescript",
       ]);
       expect(stdout.trim()).toBe(
         `💿 That's it! \`cd\` into "${projectDir}" and check the README for development and deploy instructions!`
       );
       expect(fs.existsSync(path.join(projectDir, "package.json"))).toBeTruthy();
-      expect(fs.existsSync(path.join(projectDir, "app/root.tsx"))).toBeTruthy();
+      expect(fs.existsSync(path.join(projectDir, "app/root.jsx"))).toBeTruthy();
+      expect(fs.existsSync(path.join(projectDir, "app/root.tsx"))).toBeFalsy();
+      expect(
+        fs.existsSync(path.join(projectDir, "jsconfig.json"))
+      ).toBeTruthy();
+      expect(fs.existsSync(path.join(projectDir, "tsconfig.json"))).toBeFalsy();
+      let pkgJSON = JSON.parse(
+        fs.readFileSync(path.join(projectDir, "package.json"), "utf8")
+      );
+      expect(pkgJSON.scripts.typecheck).toBeUndefined();
+      expect(pkgJSON.devDependencies).not.toContain("typescript");
```

Comment:
I believe `toContain` only works on arrays, and possibly strings. I think you'll want to use `Object.keys(pkgJSON.devDependencies)` here.

Reply:
yup, you're right https://jestjs.io/docs/expect#tocontainitem